yadro/sensors.cpp

```diff
@@ -70,15 +70,15 @@ sensor_t::state_t sensor_t::enable(bool s)
     {
         state = E_CRITICAL_HIGH;
     }
-    else if (criticalLow > 0 && currentValue > criticalLow)
+    else if (criticalLow > 0 && currentValue < criticalLow)
```

Comment:
Looks like this code assumes that the sensor value can't be negative or that the Lower Critical threshold can't be zero, which is generally not true. There may be a negative voltage sensor (e. g., -12V), or a wide range temperature sensor that can easily cross zero or have it as the lower threshold.